src/k8s/deployment.ts

```diff
@@ -71,6 +75,15 @@ export class DeploymentConfig {
             'You have no replicas available');
     }
 
+    static async dcShowLog(context: { id: any; }): Promise<string> {
```

Comment:
Should be named `showLog`. No need for 'dc' prefix, because the method belongs to DeploymentConfig class.